src/angular/schematics/ng-update/index.ts

```diff
@@ -11,32 +14,34 @@ import { ClassNamesMigration } from '../ng-add/migrations/class-names';
 import { leanTestConfigurationMigration } from './migrations/lean-test-configuration-migration';
 import { sbbAngularUpgradeData } from './upgrade-data';
 
-const sbbAngularMigrations: NullableDevkitMigration[] = [];
-
 /** Entry point for the migration schematics with target of SBB Angular v13 */
 export function updateToV13(): Rule {
+  patchUpdateBuffer();
   return chain([
     leanTestConfigurationMigration,
-    createMigrationSchematicRule(
-      TargetVersion.V13,
-      sbbAngularMigrations,
-      sbbAngularUpgradeData,
-      onMigrationComplete
-    ),
+    createMigrationSchematicRule(TargetVersion.V13, [], sbbAngularUpgradeData, onMigrationComplete),
   ]);
 }
 
 /** Entry point for the migration schematics with target of Angular CDK 14.0.0 */
 export function updateToV14(): Rule {
```

Comment:
FYI: I didn't remove this code because we may need it for renaming the `sbb-autocomplete-hint` to `sbb-option-hint`.